examples/inference/build_gptq_weight.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import logging
+
+from auto_gptq import AutoGPTQForCausalLM, BaseQuantizeConfig
+from transformers import AutoTokenizer
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)s [%(name)s] %(message)s", level=logging.INFO, datefmt="%Y-%m-%d %H:%M:%S"
+)
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--pretrained_model_dir",
+    type=str,
+    help="pretrained model directory",
+)
+parser.add_argument(
+    "--quantized_model_dir",
+    help="the path of out gptq model",
+    type=str,
+)
+args = parser.parse_args()
+pretrained_model_dir = args.pretrained_model_dir
+quantized_model_dir = args.quantized_model_dir
+
+
+quantize_config = BaseQuantizeConfig(
+    bits=4,  # quantize model to 4-bit
```

Comment:
These options should be configurable via cli

Reply:
Update.